source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java

```diff
@@ -69,6 +70,24 @@ public static void setTextSize(TextView textView, FontType type, TextSize textSi
         textView.setTextSize(TextRendererUtil.getTextSize(type, textSize, hostConfig));
     }
 
+    /**
+     * Applies given TextStyle to the given TextView
+     * @param textView TextView to apply style to
+     * @param style TextStyle to apply
+     */
+    private static void applyTextStyle(TextView textView, final TextStyle style)
+    {
+        // Indicate Heading to accessibility service
+        // TODO: Refactor to ViewCompat.setAccessibilityHeading after AndroidX upgrade
+        ViewCompat.setAccessibilityDelegate(textView, new AccessibilityDelegateCompat() {
```

Comment:
neat

Reply:
is it part of Android SDK? wish iOS has something similar, iOS VO seems to have a timing-related. bug

Reply:
UWP has something similar

https://github.com/microsoft/AdaptiveCards/blob/2257e48ae9a795ea33760d442f0a6b1523c6c682/source/uwp/Renderer/lib/LinkButton.cpp#L8-L12

Reply:
Yeah, need to use the delegate on the version of support library we're using. Newer versions have a direct property similar to iOS.